config/configs/minimal.yaml

```diff
@@ -83,7 +83,7 @@ INACTIVITY_SCORE_RECOVERY_RATE: 16
 # 2**4 * 10**9 (= 16,000,000,000) Gwei
 EJECTION_BALANCE: 16000000000
 # [customized] more easily demonstrate the difference between this value and the activation churn limit
-MIN_PER_EPOCH_CHURN_LIMIT: 2
+MIN_PER_EPOCH_CHURN_LIMIT: 4
```

Comment:
this is the constant that was breaking the minimal tests. I looked for other client consensus clients, and I think the value should be 4 now.

Reply:
Should be fixed by #328. We were using the wrong config files 👀

Reply:
#328 was merged 🚀. Please fix conflicts